hardware/camera/hqcam_filter_removal.md

```diff
@@ -6,11 +6,17 @@ The High Quality Camera contains an IR filter, which is used to reduce the camer
 
 To remove the filter:
 1. Work in a clean and dust-free environment, as the sensor will be exposed to the air.
-1. Unscrew the two 1.5 mm hex lock keys on the underside of the main circuit board. Be careful not to let the washers roll away.
+![camera sensor](rpi_hq_cam_sensor.jpg)
+1. Unscrew the two 1.5 mm hex lock keys on the underside of the main circuit board. Be careful not to let the washers roll away. There is a gasket of slightly sticky material between the housing and PCB which will require some force to separate.
+![camera gasket](rpi_hq_cam_gasket.jpg)
 1. Lift up the board and place it down on a very clean surface. Make sure the sensor does not touch the surface.
 1. Before completing the next step, read through all of the steps and decide whether you are willing to void your warranty. **Do not proceed** unless you are sure that you are willing to void your warranty.
 1. Turn the lens around so that it is "looking" upwards and place it on a table. Using a pen top or similar soft plastic item, push down on the filter. The glue will break and the filter will detach from the lens mount.
+The filter is about 1.1mm thick, glued around the perimeter - if you apply a little isopropyl alcohol and a little heat (~20-30 C), then only apply force to the very edges where the glass attaches to the aluminium - this can be done without breaking the filter.
+![camera ir filter](rpi_hq_cam_ir_filter.jpg)
```

Comment:
Perhaps "Alternatively" can be used here to indicated you can use either method a little more clearly.